### PR TITLE
[MM-13033] Continue bulk import on large image error

### PR DESCRIPTION
**Pull request by liusy182**
#### Summary

Log warning message and continue bulk import when the error is related to large image size.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/10754

### CODE REVIEW

File: app/import_functions.go [hunk 1]
```diff
@@ -923,15 +922,15 @@ func (a *App) ImportAttachment(data *AttachmentImportData, post *model.Post, tea
 
 		if err != nil {
 			fmt.Print(err)
```
Comment (gabrieljackson) [inline app/import_functions.go]: nit: completely optional improvement here, but we could switch this from using `fmt.Print` to `mlog` for some cleaner error reporting here.

**Comment by liusy182** [comment]
@gabrieljackson added test for `stopOnError()`. It is a bit hard to add tests in other areas without proper mocking or introducing large image in test artifacts. Let me know if you have better ideas and I am happy to add that.

File: app/import_functions.go [hunk 2]
```diff
@@ -922,16 +921,16 @@ func (a *App) ImportAttachment(data *AttachmentImportData, post *model.Post, tea
 		fileInfo, err := a.DoUploadFile(timestamp, teamId, post.ChannelId, post.UserId, file.Name(), buf.Bytes())
 
 		if err != nil {
-			fmt.Print(err)
-			return nil, fileUploadError
+			mlog.Error(fmt.Sprintf("Fail to upload file: %s", err.Error()))
```
Comment (gabrieljackson) [inline app/import_functions.go]:
```suggestion
			mlog.Error(fmt.Sprintf("Failed to upload file: %s", err.Error()))
```

**Comment by hanzei** [comment]
Swapping out @jespino for @grundleborg, given that Georg is OOO.

**Comment by jespino** [comment]
Thanks @liusy182 you are doing a great work 🎉

**Pull request merged**